test/e2e_env/kubernetes/meshhttproute/test.go

```diff
@@ -18,6 +18,29 @@ func Test() {
 	meshName := "meshhttproute"
 	namespace := "meshhttproute"
 
+	kubeMeshServiceYAML := fmt.Sprintf(`
```

Comment:
Could we move this test to the multizone setup? I believe the first cluster already has MeshService enabled, so we wouldn't need to create them manually.

Reply:
moved to multizone tests